src/components/common/child-entry/child-entry.styles.tsx

```diff
@@ -0,0 +1,32 @@
+import { Color } from "@lifesg/react-design-system/color";
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    background: ${Color.Neutral[7]};
+    display: flex;
+    flex-direction: row;
+    width: 100%;
```

Comment:
- remove `flex-direction` as it is row by default
- replace width with `flex: 1`